### ISSUE TITLE
No focus indicator on status bar item that has no command tied to it in the status bar.

### ISSUE THREAD

**Issue by miniwangdali** [microsoft/vscode#203852]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.85.2
- OS Version: Windows 11 Enterprise (Windows_NT x64 10.0.22631.3007)

```
Version: 1.85.2 (user setup)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:10.514Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
```

**Description**
We are making a VS Code extension. Our extension leverages VS Code status bar to show some sync status of the language server. During the accessibility test, the tester reported that one of our status item has no focus indicator. This only happens when a status bar item has no command set to it. I checked html of that item. it seems there is a `disabled` class on the anchor element. Is it by design to have disable class on the items that has no command? or should we get a fix?
![image](https://github.com/microsoft/vscode/assets/5415452/1c429e9b-a0f8-481a-96a3-a8139bf5c020)

https://github.com/microsoft/vscode/assets/5415452/e14b590d-0d9c-4328-95fd-34eaa59dc16e

**Comment by meganrogge** [comment]
This is by design. You should add a command.

**Comment by nima-ap** [comment]
Why is it the case that a status bar item _must_ have a command to get a focus indicator?
What if it is purely informational and does not _need_ a command?
